scripts/powershell/sub-cleanup/sub-cleanup.ps1

```diff
@@ -34,7 +34,13 @@ param(
   [string]$Subscription,
   
   [Parameter(Mandatory = $false)]
-  [boolean]$Audit=$true
+  [boolean]$Audit=$true,
```

Comment:
@heaths we're going to refactor this script into the main live test cleanup script, at which point I'll update the parameters to switch. I didn't change them in my PR to @kurtzeborn's branch because I didn't want to update how he was used to invoking the script.